Gems/ROS2/Code/Source/RobotImporter/ROS2RobotImporterEditorSystemComponent.cpp

```diff
@@ -173,12 +195,22 @@ namespace ROS2
             }
         };
 
-        AZStd::string prefabName = AZStd::string(parsedUrdf->getName().c_str(), parsedUrdf->getName().size()) + ".prefab";
+        uint64_t urdfWorldCount = parsedUrdfRoot.WorldCount();
+        if (urdfWorldCount == 0)
+        {
```

Comment:
I am thinking we could check if there is at least 1 model, otherwise the URDF would have any robot data within it.

Reply:
Well we use the Model name or World name for the name of prefab that gets generated. 
If want we can use the name of the URDF file itself and then let the URDFPrefabMaker code take of dealing with not having any models.

Reply:
Yeah, I noticed that is was the case for URDF conversion.
It comes with only a root <model> without any <world> tags within it.

I have updated the logic, to use the name of the root <model> tag if it exist, otherwise it uses the name of first <world> tag as the prefab name.